### PR TITLE
Replaced egrep with grep -E. bsc#1203227

### PR DESCRIPTION
**Pull request by mchf**
## Problem

See https://github.com/yast/yast.github.io/pull/336 for details

### CODE REVIEW

File: po/auth-client/nl.po [hunk 1]
```diff
@@ -1675,7 +1675,7 @@ msgstr ""
 msgid ""
 "The LDAP attribute that contains the protocols understood by this service."
 msgstr ""
-"Het LDAP-attribuut dat de protocollen bevat die worden begrepen door deze "
+"Het LDAP-attribuut dat de protocollen bevat die worden bgrep -Een door deze "
```
Comment (mvidner) [inline po/auth-client/nl.po]: :heart_eyes: :heart_eyes: :heart_eyes: I win! !! :exclamation: :heavy_heart_exclamation:

File: po/storage/sv.po [hunk 2]
```diff
@@ -12878,7 +12878,7 @@ msgstr "Klicka nedan för att se mer information (endast engelska)."
 #~ msgstr ""
 #~ "<p>\n"
 #~ "Den här dialogrutan är till för <b>experter</b>.\n"
-#~ "Om du inte vet vad begreppet <b>partition</b> innebär,\n"
+#~ "Om du inte vet vad bgrep -Epet <b>partition</b> innebär,\n"
```
Comment (mvidner) [inline po/storage/sv.po]: Ahem

**Review by mvidner — CHANGES_REQUESTED** [review]
Now we know that Dutch and Swedish will never deprecate `egrep`... 

The rest is just translators comment but I guess it is fine to fix it there.

File: po/auth-client/nl.po (hunk 1 above)
Comment (mchf) [reply po/auth-client/nl.po]: nice one ;-)

**Review by mvidner — APPROVED** [review]

**Pull request merged**